app/models/show.rb

```diff
@@ -82,5 +82,33 @@ def duration
       return "#{h.strip}h #{m}min"
     end
   end
+
+  def dates
+    dates = self.performances.map do |performance|
+      if performance.start.to_datetime < performance.start.to_datetime.beginning_of_day + 0.25
+      performance.start.day - 1
+      else
+        performance.start.day
+      end
+    end.uniq
```

Comment:
What does the end.uniq do?

Reply:
.uniq makes sure any duplicates are excluded (coz some shows have more than one performance a day so i wanna remove the duplicate entries from the data before i start creating the string)